inkscape figure conversion in the manual

Either copy files to build with CMake or wait long enough for inkscape to fix this (or disable the command line interface)

Ok, I confirmed that:
```
[ 91%] Generating fig/logo.eps
cd /builddir/build/BUILD/votca-1.5/s390x-redhat-linux-gnu/ctp/manual/fig && /usr/bin/inkscape -f /builddir/build/BUILD/votca-1.5/ctp/manual/fig/logo.svg -E /builddir/build/BUILD/votca-1.5/s390x-redhat-linux-gnu/ctp/manual/fig/logo.eps
make[2]: Leaving directory '/builddir/build/BUILD/votca-1.5/s390x-redhat-linux-gnu'
BUILDSTDERR: terminate called after throwing an instance of 'std::logic_error'
BUILDSTDERR:   what():  basic_string::_M_construct null not valid
BUILDSTDERR: Emergency save activated!
BUILDSTDERR: Emergency save completed. Inkscape will close now.
BUILDSTDERR: If you can reproduce this crash, please file a bug at www.inkscape.org
BUILDSTDERR: with a detailed description of the steps leading to the crash, so we can fix it.
BUILDSTDERR: ** Message: 05:52:41.967: Error: Inkscape encountered an internal error and will close now.
BUILDSTDERR: make[2]: *** [ctp/manual/CMakeFiles/ctp-manual_dvi.dir/build.make:145: ctp/manual/fig/logo.eps] Aborted (core dumped)
BUILDSTDERR: make[2]: *** Waiting for unfinished jobs....
```
in https://koji.fedoraproject.org/koji/taskinfo?taskID=32404147

--- a/manual/fig/CMakeLists.txt
+++ b/manual/fig/CMakeLists.txt
@@ -1,8 +1,9 @@
 foreach(PIC fig/logo fig/dcv2t fig/coupling fig/fragment_segment fig/monomer_parabolas fig/workflow)
   add_custom_command(OUTPUT ${PIC}.eps
-    COMMAND ${INKSCAPE} -f ${CMAKE_CURRENT_SOURCE_DIR}/${PIC}.svg
-      -E ${CMAKE_CURRENT_BINARY_DIR}/${PIC}.eps
-    WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/fig
+    COMMAND ${CMAKE_COMMAND} -E copy_if_different ${CMAKE_CURRENT_SOURCE_DIR}/${PIC}.svg
+        ${CMAKE_CURRENT_BINARY_DIR}/${PIC}.svg
+    COMMAND ${INKSCAPE} -f ${PIC}.svg -E ${PIC}.eps
+    WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
     DEPENDS ${PIC}.svg)
   list(APPEND manual_FIGURES "${PIC}.eps")
 endforeach()

